terraform/modules/self-service/modules/cognito/cognito.tf

```diff
@@ -2,9 +2,94 @@ locals {
   service = "self-service"
 }
 
+resource "aws_sns_topic" "cognito-sns-topic" {
+  name = "cognito-sns-topic"
+}
+
+resource "aws_sns_topic_policy" "cognito-sns-policy" {
+  arn = "${aws_sns_topic.cognito-sns-topic.arn}"
+
+  policy = "${data.aws_iam_policy_document.sns-topic-policy.json}"
+}
+
+data "aws_iam_policy_document" "sns-topic-policy" {
+  policy_id = "__default_policy_ID"
+
+  statement {
+    actions = [
+      "SNS:Publish"
+    ]
+
+    effect = "Allow"
+
+    principals {
+      type        = "AWS"
+      identifiers = ["*"]
+    }
+
+    resources = [
+      "${aws_sns_topic.cognito-sns-topic.arn}",
+    ]
+
+    sid = "__default_statement_ID"
```

Comment:
is this required? I believe this is an arbitrary text, so we could use something more descriptive..

Reply:
Put `20190910-cognito-sns-policy` in the SID